src/class_sqlThread.py

```diff
@@ -17,128 +19,107 @@
+                sql_as_string = sql_file.read()
+            self.cur.executescript(sql_as_string)
+        except OSError as err:
+            logger.debug('The file is missing. Error message: %s\n', str(err))
+            return
+        except IOError as err:
+            logger.debug(
+                'ERROR trying to initialize database. Error message: %s\n', str(err))
         except Exception as err:
-            if str(err) == 'table inbox already exists':
-                logger.debug('Database file already exists.')
-
-            else:
-                sys.stderr.write(
-                    'ERROR trying to create database file (message.dat). Error message: %s\n' % str(err))
-                os._exit(0)
-
-        # If the settings version is equal to 2 or 3 then the
-        # sqlThread will modify the pubkeys table and change
-        # the settings version to 4.
-        settingsversion = config.getint(
-            'bitmessagesettings', 'settingsversion')
-
-        # People running earlier versions of PyBitmessage do not have the
-        # usedpersonally field in their pubkeys table. Let's add it.
-        if settingsversion == 2:
-            item = '''ALTER TABLE pubkeys ADD usedpersonally text DEFAULT 'no' '''
-            parameters = ''
-            self.cur.execute(item, parameters)
-            self.conn.commit()
-
-            settingsversion = 3
-
-        # People running earlier versions of PyBitmessage do not have the
-        # encodingtype field in their inbox and sent tables or the read field
-        # in the inbox table. Let's add them.
-        if settingsversion == 3:
-            item = '''ALTER TABLE inbox ADD encodingtype int DEFAULT '2' '''
-            parameters = ''
-            self.cur.execute(item, parameters)
-
-            item = '''ALTER TABLE inbox ADD read bool DEFAULT '1' '''
-            parameters = ''
-            self.cur.execute(item, parameters)
-
-            item = '''ALTER TABLE sent ADD encodingtype int DEFAULT '2' '''
-            parameters = ''
-            self.cur.execute(item, parameters)
-            self.conn.commit()
-
-            settingsversion = 4
-
-        config.set(
-            'bitmessagesettings', 'settingsversion', str(settingsversion))
-        config.save()
-
-        helper_startup.updateConfig()
-
-        # From now on, let us keep a 'version' embedded in the messages.dat
-        # file so that when we make changes to the database, the database
-        # version we are on can stay embedded in the messages.dat file. Let us
-        # check to see if the settings table exists yet.
-        item = '''SELECT name FROM sqlite_master WHERE type='table' AND name='settings';'''
-        parameters = ''
-        self.cur.execute(item, parameters)
+            logger.debug(
+                'ERROR trying to initialize database. Error message: %s\n', str(err))
+
```

Comment:
we seem to be missing a sqlite error handler. This should also log, but stronger severity (e.g. `error`), not `debug`